src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.cs

```diff
@@ -39,7 +39,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics
     internal abstract partial class AbstractDiagnosticsTaggerProvider<TTag> : AsynchronousTaggerProvider<TTag>
         where TTag : ITag
     {
-        private readonly IDiagnosticService _diagnosticService;
+        protected readonly IDiagnosticService DiagnosticService;
```

Comment:
why?

Reply:
Does not need to be changed, it was probably from some old commit. Changed back